ODSBox AMI build failed due to a installation error of chrome package

The ODSBox AMI build fails due to a installation error of chrome package.

Error: 
``` 
03:13:38  ==> amazon-ebs: Error: Package: google-chrome-stable-95.0.4638.54-1.x86_64 (/google-chrome-stable_current_x86_64)
03:13:38  ==> amazon-ebs:            Requires: libc.so.6(GLIBC_2.18)(64bit)
03:13:51      amazon-ebs:  You could try running: rpm -Va --nofiles --nodigest
03:13:51  2021/10/20 01:13:50 [INFO] 667766 bytes written for 'stdout'
03:13:51  2021/10/20 01:13:50 packer-builder-amazon-ebs plugin: [ERROR] Remote command exited with '1': /tmp/script_1364.sh  --branch 4.x
03:13:51  2021/10/20 01:13:50 packer-builder-amazon-ebs plugin: [INFO] RPC endpoint: Communicator ended with: 1
03:13:51  2021/10/20 01:13:50 [INFO] 9064 bytes written for 'stderr'
``` 

This is impacting the build of the ODSBox AMI for all mayor branches: master, 4,x, 3,x

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -33,7 +33,8 @@
 - ds-jupyter-notebook renamed to ds-jupyter-lab and upgrade to JupyterLab 3 ([#562](https://github.com/opendevstack/ods-quickstarters/issues/562))
 
 ### Fixed
-- ODS AMI build failed due to  to jira missing permissions on jira data folder ([#1005](https://github.com/opendevstack/ods-core/pull/1005))
+- ODS AMI build failed due to an installation error of chrome package ([#1054](https://github.com/opendevstack/ods-core/pull/1054))
+- ODS AMI build failed due to jira missing permissions on jira data folder ([#1005](https://github.com/opendevstack/ods-core/pull/1005))
 - ODS AMI build failed due to bitbucket crashed container ([#1001](https://github.com/opendevstack/ods-core/pull/1001))
 - Preserve clusterIPs of services ([#983](https://github.com/opendevstack/ods-core/pull/983))
 - Use storageClassName instead of annotation ([#985](https://github.com/opendevstack/ods-core/pull/985))

--- a/ods-devenv/scripts/deploy.sh
+++ b/ods-devenv/scripts/deploy.sh
@@ -373,10 +373,15 @@ EOF
 #######################################
 function setup_google_chrome() {
     if [[ -z $(command -v google-chrome) ]]
+
+    chrome_version=94.0.4606.81
+
     then
-        curl -LO https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm
-        sudo yum install -y ./google-chrome-stable_current_*.rpm
-        rm ./google-chrome-stable_current_*.rpm
+        echo "Download and install chrome (google-chrome-stable-${chrome_version}-1.x86_64.rpm)!"
+        curl -LO https://dl.google.com/linux/chrome/rpm/stable/x86_64/google-chrome-stable-${chrome_version}-1.x86_64.rpm
+        sudo yum install -y ./google-chrome-stable-${chrome_version}-1.x86_64.rpm
+        rm ./google-chrome-stable-${chrome_version}-1.x86_64.rpm
+        echo "... chrome installation completed!"
     fi
 }
 

